mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -6850,6 +6892,50 @@ resource "google_compute_instance" "foobar" {
 `, instance, enableConfidentialCompute)
 }
 
+<% unless version == 'ga' -%>
+func testAccComputeInstanceConfidentialHyperDiskBootDisk(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_compute_image" "my_image" {
+  family    = "ubuntu-2204-lts"
+  project   = "ubuntu-os-cloud"
+}
+
+data "google_project" "project" {}
+
+resource "google_kms_crypto_key_iam_binding" "crypto_key" {
```

Comment:
@melinath did you ensure we weren't relying on other permissions on this role? This should have been a member, not a binding.

Reply:
It looks like this is a pretty common pattern for our tests. There are some that use member, but a number use binding. But they should all be using BootstrapPSARole instead of a resource.

Reply:
That attaches to a resource provisioned outside of the test? Tests can use a member if they create the service account they attach to the target resource inside of the test. Bindings/policies should only be used if the target resource is created within the test.

Reply:
ah, yes, you're correct. For some reason I thought this was at the project level but it's on the specific key. In that case, though, it shouldn't matter whether it uses member or binding.

Reply:
Nah, this should bootstrap regardless- it's created outside of the test (we can't guarantee it doesn't get reused) and the service account is a preexisting one. If the service account was created here we could have used a member.

Since the bootstrapped key does look new we're unlikely to have overwritten any permissions on it.

Reply:
I dont mind rewriting the test @rileykarson  to use  member instead of binding

Reply:
@NA2047 it should use bootstrapping with BootstrapPSARole rather than using a member or binding.

Reply:
This case should have used a bootstrapped role, yep- sorry for the confusion, I was highlighting that binding is _definitely_ not safe for a case like this. member could have been if we made our own SA in the test.